backend/src/main/resources/application-local.yml

```
@@ -10,7 +10,7 @@ spring:
       hibernate:
         format_sql: true
   datasource:
-    url: jdbc:mysql://localhost:3306/ternoko?serverTimezone=Asia/Seoul
+    url:
     username: root
     password:
   h2:
```

local은 h2로 돌리기로 했어서 해당 yml 설정은 바꾸면 안될 것 같아요
각자 로컬에 mysql 설치하고 `create database ternoko;`만 해주시면 로컬로 돌아갈거에요!
안되시면 말씀해주세요! 리드미에도 적어놔야겠네용

```suggestion
    url: jdbc:mysql://localhost:3306/ternoko?serverTimezone=Asia/Seoul
    username: root
    password:
```